github/ci/prow-deploy/files/jobs/k8snetworkplumbingwg/ovs-cni/ovs-cni-presubmits-0.29.yaml

```diff
@@ -0,0 +1,33 @@
+---
+presubmits:
+  k8snetworkplumbingwg/ovs-cni:
+    - name: pull-e2e-ovs-cni
```

Comment:
We need to state the version at job name to prevent prow from getting confuse

Reply:
Ah, good catch. Should be fixed now

Reply:
suggestion: you can add `context: pull-e2e-ovs-cni` here to map to the same GitHub status

Reply:
Aha! I did not know. Let me add it. Is it ok to add it to the main branch too? (So I would copy-paste it while branching some future release)

Reply:
If you are using the [config-forker from `kubernetes/test-infra`](https://github.com/kubernetes/test-infra/tree/master/releng/config-forker#readme) it will add the context automatically IIRC.

Reply:
Gotcha. I would usually just copy the file and change a few fields than try to recall how to use the forker :grimacing:

Reply:
> Gotcha. I would usually just copy the file and change a few fields than try to recall how to use the forker grimacing

TBH if we hadn't automated this, I'd be doing it the same way :)